lib/rules/validate-jsdoc/disallow-newline-after-description.js

```diff
@@ -23,10 +23,5 @@ function disallowNewlineAfterDescription(node, err) {
         return;
     }
 
-    var loc = node.jsdoc.loc.start;
-    var lines = doc.description.split(RE_NEWLINES);
-    err('Newline required after description', {
-        line: loc.line + lines.length,
-        column: loc.column + 3 + lines[lines.length - 1].length
-    });
+    err('Newline required after description', doc.tags[0].loc.offset - 4);
```

Comment:
Temporary calculated) Since you don't maintain description offsets, it's tricky to put correct number here.